sdk/eventhub/event-hubs/src/connectionContext.ts

```diff
@@ -203,27 +205,27 @@ export namespace ConnectionContext {
         // then the rhea connection is in the process of terminating.
         return Boolean(!this.connection.isOpen() && this.connection.isRemoteOpen());
       },
-      async readyToOpenLink() {
+      async readyToOpenLink(optionsArg?: { abortSignal?: AbortSignalLike }) {
```

Comment:
why not just call it `options`?

Reply:
Same name exists in outer scope and the linter wasn’t happy about it.